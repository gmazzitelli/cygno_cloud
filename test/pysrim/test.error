0009:err:ole:CoGetClassObject class {f9043c85-f6f2-101a-a3c9-08002b2f49fb} not registered
0009:err:ole:CoGetClassObject class {f9043c85-f6f2-101a-a3c9-08002b2f49fb} not registered
0009:err:ole:CoGetClassObject no class object {f9043c85-f6f2-101a-a3c9-08002b2f49fb} could be created for context 0x3
0009:err:ole:CoGetClassObject class {6262d3a0-531b-11cf-91f6-c2863c385e30} not registered
0009:err:ole:CoGetClassObject class {6262d3a0-531b-11cf-91f6-c2863c385e30} not registered
0009:err:ole:CoGetClassObject no class object {6262d3a0-531b-11cf-91f6-c2863c385e30} could be created for context 0x3
XIO:  fatal IO error 11 (Resource temporarily unavailable) on X server ":101"
      after 207 requests (183 known processed) with 0 events remaining.
XIO:  fatal IO error 11 (Resource temporarily unavailable) on X server ":101"
      after 63 requests (63 known processed) with 0 events remaining.
