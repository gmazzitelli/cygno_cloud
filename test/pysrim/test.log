000 (162.000.000) 2023-01-23 14:35:41 Job submitted from host: <131.154.96.235:31618?addrs=131.154.96.235-31618&alias=131.154.96.235.myip.cloud.infn.it&noUDP&sock=schedd_22_ab2c>
...
040 (162.000.000) 2023-01-23 14:36:01 Started transferring input files
	Transferring to host: <10.244.3.55:9618?CCBID=131.154.96.235:30618%3faddrs%3d131.154.96.235-30618%26alias%3dccb-pod-7bb6bddd45-v242f%26noUDP%26sock%3dcollector#31&PrivNet=wn-pod-74b4fb5979-mxbh6&addrs=10.244.3.55-9618&alias=wn-pod-74b4fb5979-mxbh6&noUDP&sock=slot1_1_57_e95b_32>
...
040 (162.000.000) 2023-01-23 14:36:01 Finished transferring input files
...
001 (162.000.000) 2023-01-23 14:36:02 Job executing on host: <10.244.3.55:9618?CCBID=131.154.96.235:30618%3faddrs%3d131.154.96.235-30618%26alias%3dccb-pod-7bb6bddd45-v242f%26noUDP%26sock%3dcollector#31&PrivNet=wn-pod-74b4fb5979-mxbh6&addrs=10.244.3.55-9618&alias=wn-pod-74b4fb5979-mxbh6&noUDP&sock=startd_24_0a5c>
...
006 (162.000.000) 2023-01-23 14:36:03 Image size of job updated: 10092
	10  -  MemoryUsage of job (MB)
	10092  -  ResidentSetSize of job (KB)
...
040 (162.000.000) 2023-01-23 14:36:03 Started transferring output files
...
040 (162.000.000) 2023-01-23 14:36:03 Finished transferring output files
...
005 (162.000.000) 2023-01-23 14:36:04 Job terminated.
	(1) Normal termination (return value 1)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	718  -  Run Bytes Sent By Job
	7114  -  Run Bytes Received By Job
	718  -  Total Bytes Sent By Job
	7114  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1         1 
	   Disk (KB)            :       19       10      8873 
	   Memory (MB)          :       10        1       128 

	Job terminated of its own accord at 2023-01-23T14:36:03Z.
...
