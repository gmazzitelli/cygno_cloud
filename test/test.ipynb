{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "novel-stereo",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Welcome to JupyROOT 6.24/08\n"
     ]
    }
   ],
   "source": [
    "import sys\n",
    "import ROOT\n",
    "import root_numpy as rtnp\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "verbal-pursuit",
   "metadata": {},
   "outputs": [
    {
     "ename": "OSError",
     "evalue": "Failed to open file https://swift.cloud.infn.it:8080/v1/AUTH_1e60fe39fba04701aa5ffc0b97871ed8/Cygnus/Data/LAB/histograms_Run04037.root",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mOSError\u001b[0m                                   Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-2-7c06f3f68202>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mf\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mROOT\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mTFile\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mOpen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'https://swift.cloud.infn.it:8080/v1/AUTH_1e60fe39fba04701aa5ffc0b97871ed8/Cygnus/Data/LAB/histograms_Run04037.root'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0mprint\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0;34m'Find Keys: '\u001b[0m\u001b[0;34m+\u001b[0m\u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mf\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mGetListOfKeys\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib64/python3.6/site-packages/ROOT/pythonization/_tfile.py\u001b[0m in \u001b[0;36m_TFileOpen\u001b[0;34m(klass, *args)\u001b[0m\n\u001b[1;32m     65\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mf\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mbind_object\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mklass\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     66\u001b[0m         \u001b[0;31m# args[0] can be either a string or a TFileOpenHandle\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 67\u001b[0;31m         \u001b[0;32mraise\u001b[0m \u001b[0mOSError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'Failed to open file {}'\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mformat\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     68\u001b[0m     \u001b[0;32mreturn\u001b[0m \u001b[0mf\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     69\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mOSError\u001b[0m: Failed to open file https://swift.cloud.infn.it:8080/v1/AUTH_1e60fe39fba04701aa5ffc0b97871ed8/Cygnus/Data/LAB/histograms_Run04037.root"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error in <DavixOpen>: can not open file \"https://swift.cloud.infn.it:8080/v1/AUTH_1e60fe39fba04701aa5ffc0b97871ed8/Cygnus/Data/LAB/histograms_Run04037.root\" with davix: Connection timed out (8)\n"
     ]
    }
   ],
   "source": [
    "f=ROOT.TFile.Open('https://swift.cloud.infn.it:8080/v1/AUTH_1e60fe39fba04701aa5ffc0b97871ed8/Cygnus/Data/LAB/histograms_Run04037.root')\n",
    "print ('Find Keys: '+str(len(f.GetListOfKeys())))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "dependent-facility",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "echo oidc-keychain: Agent pid 138\n",
      ";\n",
      "OIDC_SOCK=/tmp/oidc-owanxz/oidc-agent.133; export OIDC_SOCK;\n",
      "OIDCD_PID=138; export OIDCD_PID;\n",
      "\u001b[31mCould not get the socket path from env var 'OIDC_REMOTE_SOCK'. Have you set the env var?\n",
      "\u001b[0m\u001b[31mError: Env var not set\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! oidc-keychain\n",
    "import os\n",
    "os.environ['OIDC_SOCK']='/tmp/oidc-WkcYuk/oidc-agent.118'\n",
    "os.environ['OIDCD_PID']='123'\n",
    "! oidc-token infncloud"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "equipped-brown",
   "metadata": {},
   "outputs": [],
   "source": [
    "im"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
