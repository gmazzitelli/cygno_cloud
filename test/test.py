#!/usr/bin/env python3.6
import os

for name, value in os.environ.items():
    print("{0}: {1}".format(name, value))

