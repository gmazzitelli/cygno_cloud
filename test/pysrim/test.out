---- running: 
win32, /var/lib/condor/.wine
wine: failed to initialize: libm.so.6: failed to map segment from shared object
The layer width is 57.0 microns.
Running event from He of ion 0 for the energy of 1 keV.
Traceback (most recent call last):
  File "/usr/local/srim/pysrim_test.py", line 64, in <module>
    results = trim.run(srim_executable_directory)
  File "/usr/local/lib/python3.6/dist-packages/srim/srim.py", line 219, in run
    subprocess.check_call(['wine', str(os.path.join('.', 'TRIM.exe'))])
  File "/usr/lib/python3.6/subprocess.py", line 311, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['wine', './TRIM.exe']' returned non-zero exit status 1.
